XRTK-Core/Packages/com.xrtk.core/Utilities/BuildAndDeploy/LuminPlayerBuildTools.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) XRTK. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Diagnostics;
+using UnityEditor.Build.Reporting;
+using UnityEngine;
+using XRTK.Extensions;
+
+namespace XRTK.Utilities.Build
+{
+    /// <summary>
+    /// Class containing various utility methods to build a lumin mpk installer from a Unity project.
+    /// </summary>
+    public class LuminPlayerBuildTools
+    {
+        /// <summary>
+        /// Build the Lumin Player.
+        /// </summary>
+        /// <param name="buildInfo"></param>
+        public static BuildReport BuildPlayer(BuildInfo buildInfo)
+        {
+            if (!Application.isBatchMode)
+            {
+                buildInfo.PostBuildAction += PostBuildAction;
+            }
+
+            return UnityPlayerBuildTools.BuildUnityPlayer(buildInfo);
```

Comment:
Would this not be better implemented as an extension class, calling base.UnityPlayerBuildTools, rather than this complex interweving.  

Lumin should be an override of the base rather than separate "bits"

Reply:
Agreed, or maybe needs an interface for the build tools that then calls the right method for the right build platform

Reply:
Looked into this more. Because the `UnityPlayerBuildTools` class is static and is getting methods called directly by the editor, we cannot use derived classes to correctly call the appropriate overriding method.